packages/component/src/components/app-sidebar/sidebar-header/index.tsx

```diff
@@ -29,33 +35,47 @@ export const SidebarHeader = (props: SidebarHeaderProps) => {
     >
       <SidebarSwitch show={open} />
       <div className={navHeaderNavigationButtons}>
-        <IconButton
-          className={navHeaderButton}
-          data-testid="app-sidebar-arrow-button-back"
-          disabled={props.router?.history.current === 0}
-          onClick={() => {
-            props.router?.back();
-          }}
+        <Tooltip
+          content={`${t['com.affine.keyboardShortcuts.goBack']()}
+            ${isMacos ? '⌘ + [' : 'Ctrl + ['}
```

Comment:
Cloud we get those tips text from 'apps/core/src/hooks/use-shortcut-commands.ts' or some where but one place, not multiple same code.

Reply:
it is not a big problem, but better